n_t/Snakefile

```diff
@@ -58,6 +59,16 @@ rule homo_sapiens_Gutenkunst_stairwayplot_plot:
         "homo_sapiens_Gutenkunst/{seeds}.{chrms}.estimated_Ne.png"
     run: plots.plot_stairway_Ne_estimate(input[0], output[0])
 
+def ne_files(wildcards):
+    return(expand("homo_sapiens_Gutenkunst/{seeds}.{chrms}.estimated_Ne.txt",
```

Comment:
Just FYI, the brackets on return aren't necessary here, and are usually omitted.